client/src/AppBundle/Service/Mailer/MailFactory.php

```diff
@@ -90,30 +93,51 @@ private function generateAbsoluteLink($area, $routeName, array $params = [])
     public function createActivationEmail(User $user)
     {
         $area = $this->getUserArea($user);
-        $homepageURL = $this->generateAbsoluteLink($area, 'homepage');
 
-        $viewParams = [
-            'name'             => $user->getFullName(),
-            'domain'           => $homepageURL,
-            'link'             => $this->generateAbsoluteLink($area, 'user_activate', [
+        $parameters = [
+            'activationLink' => $this->generateAbsoluteLink($area, 'user_activate', [
                 'action' => 'activate',
                 'token'  => $user->getRegistrationToken(),
             ]),
-            'tokenExpireHours' => User::TOKEN_EXPIRE_HOURS,
-            'homepageUrl'      => $homepageURL,
-            'recipientRole' => self::getRecipientRole($user)
+            'registerLink' => $this->generateAbsoluteLink($area, 'register'),
         ];
 
-        $email = new ModelDir\Email();
+        $email = (new ModelDir\Email())
+            ->setFromEmailNotifyID(self::NOTIFY_FROM_EMAIL_ID)
+            ->setFromName($this->translate('activation.fromName'))
+            ->setToEmail($user->getEmail())
+            ->setToName($user->getFullName())
```

Comment:
Did we decide we don't need to set this now?

Reply:
Good point, fixed